lib/schnorr.py

```diff
@@ -130,6 +130,9 @@ def sign(privkey, message_hash):
     `message_hash` should be the 32 byte sha256d hash of the tx input (or
     message) you want to sign
     '''
+   
+    if ndata is not None:
+       assert len(bytearray(ndata)) == 32
```

Comment:
Hmm -- why cast to `bytearray`?  Not sure why that's done -- the expectation is it's a `bytes` object, yes?

Reply:
i couldnt reproduce the issue i had before.  fixed in 060ca8f.